src/Component.php

```diff
@@ -62,17 +62,17 @@ public function render( array $data ) {
 	}
 
 	/**
-	 * @param string $template HTML
+	 * @param string $html HTML
 	 *
 	 * @return DOMDocument
 	 */
-	private function parseHtml( $template ) {
+	private function parseHtml( $html ) {
 		$entityLoaderDisabled = libxml_disable_entity_loader( true );
 		$internalErrors = libxml_use_internal_errors( true );
-		$document = new DOMDocument();
+		$document = new DOMDocument( null, 'UTF-8' );
```

Comment:
Probalbly version "1.0" is the righter thing™ to do, albeit the outcome will be the same.